### DISCUSSION TITLE
Környezetek elérhetősége, határidők változása

### DISCUSSION THREAD

**Discussion by Amn3s1a2018** [nav-gov-hu/eRECEIPT#44] [Kérdések és válaszok]
Az előadáson bemutatott ütemterv szerint 2024 második félévtől üzembe helyezhető ePG, de a fejlesztői dokumentáció nem tartalmazza környezetek elérhetőségeit. Kérdésem, hogy ezek mikortól lesznek használhatók? A NAV által kiadott vevői app használható lesz a teszt környezetben? Ez mikor válható?

Az előadáson megismert ütemterv érvényes még, vagy történt benne változás? Az ÁFA tv. (értelmezésem szerint erre vonatkozó) már elfogadott változása 2025 januártól lép hatályba. Ez lesz a bevezetés új időpontja?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésre.
